tests/openvino/native/quantization/test_sanity.py

```diff
@@ -29,7 +29,7 @@
     ('resnet-18-pytorch', 'imagenette2-320', {'accuracy@top1': '0.784', 'accuracy@top5': '0.946'}),
     ('mobilenet-v3-small-1.0-224-tf', 'imagenette2-320', {'accuracy@top1': '0.733', 'accuracy@top5': '0.913'}),
     ('googlenet-v3-pytorch', 'imagenette2-320', {'accuracy@top1': '0.915', 'accuracy@top5': '0.994'}),
-    ('mobilefacedet-v1-mxnet', 'wider', {'map': '0.7769245134847838'}),
+    ('mobilefacedet-v1-mxnet', 'wider', {'map': '0.7769248380270294'}),
```

Comment:
Why did you change reference metric?
I guess the test should pass with abs=0.006: `0.7769248380270294 == pytest.approx(0.7769245134847838, abs=0.006)`